### ISSUE TITLE
[NodeJS] Where is "BookInfo" ?

### ISSUE THREAD

**Issue by wooty9** [BitgetLimited/v3-bitget-api-sdk#12]
Need "BookInfo"
![image](https://user-images.githubusercontent.com/29653716/170803830-cd24b591-4d9f-4aaa-9afc-2c42bcde17eb.png)

**Comment by subz390** [comment]
yeah lots missing from this SDK, do you know what BookInfo does?  What does it relate to in the API docs?
